pkg/core/capability/caldera/connection.go

```diff
@@ -233,3 +239,64 @@ func (cc calderaConnection) GetFactSource(factSourceId string) (*models.PartialS
 
 	return response.GetPayload(), nil
 }
+
+// GetStaleOperations provides a slice of all Operations that are - based on their name -
+// created by SOARCA and are finished. These operations should in principle be deleted.
+// Unfortunately, Caldera does not allow this type of filtering to take place server-side.
+func (cc calderaConnection) GetStaleOperations() ([]*models.PartialOperation, error) {
+	response, err := cc.instance.send.Operationsops.GetAPIV2Operations(
```

Comment:
This doesn't seem to filter on the ExecutionId? I thought we wanted to do this to allow concurrent use of the Caldera instance by different SOARCA playbooks

Reply:
We find all stale (= finished && startsWith('soarca')) operations, including previous ones that remained during e.g. a crash. Then, by the name of the operation which contains the execution ID all subsequent deletes are scoped on execution ID. So this allows perfectly fine for concurrent use of the instance (since playbooks in progress will not have _finished_ operations).